app/src/main/java/org/kiwix/kiwixmobile/localFileTransfer/ReceiverDevice.kt

```diff
@@ -72,7 +72,9 @@ internal class ReceiverDevice(private val wifiDirectManager: WifiDirectManager)
                   publishProgress(fileItemIndex, FileItem.FileStatus.SENT)
                 }
               } catch (e: IOException) {
-                Log.e(TAG, e.message)
+                e.message?.let { message ->
+                  Log.e(TAG, message)
```

Comment:
A string identifier can be added to this message some like "Unable to do something: $message" also pass exception as the 3rd param to `Log.e`